src/context/meta-tags-context.php

```diff
@@ -494,6 +494,10 @@ public function generate_schema_article_type() {
 			$additional_type = $this->options->get( 'schema-article-type-' . $this->indexable->object_sub_type );
 		}
 
+		if ( stripos( $additional_type, 'Article' ) !== false ) {
```

Comment:
Would you be so kind to doc this?

Reply:
Done, also made sure the filter runs on it.